LocalisationAnalyser.Tools/Program.cs

```diff
@@ -93,7 +93,7 @@ private static async Task projectToResX(FileInfo projectFile, DirectoryInfo? out
 
                 string targetDirectory = output != null ? output.FullName : Path.GetDirectoryName(file.FilePath)!;
                 if (file.Folders.Count > 1)
-                    targetDirectory += Path.DirectorySeparatorChar + string.Join(Path.DirectorySeparatorChar, file.Folders.Skip(1));
+                    targetDirectory += string.Join(Path.DirectorySeparatorChar, file.Folders.Skip(1));
```

Comment:
Use `Path.Join` instead:

```suggestion
                    targetDirectory = Path.Join(targetDirectory, file.Folders.Skip(1));
```

(make sure to re-test, i have not tested but it should work fine)